### DISCUSSION TITLE
Data links: Add autocomplete to "title" field

### DISCUSSION THREAD

**Discussion by belm0** [grafana/grafana#38183] [Dashboards, Panels & Visualizations]
**What happened**:
try accessing autocomplete using `$` in the data link "title" field

this applies to data links in panel, fields, etc.

![Screen Shot 2020-12-01 at 2 03 05 PM](https://user-images.githubusercontent.com/1708631/100699304-f835da80-33dd-11eb-9548-db81505e93fa.png)

**What you expected to happen**:
since variable expansion is supported in the data link "title" field (not just "URL" field), I expect autocomplete to be available

**Environment**:
- Grafana version: 7.3.4

**Comment by ivanahuckova** [comment]
@grafana/grafana-frontend-platform  I wasn't sure if this is bug or feature request, but decided to add `feature-request` label as this wasn't supported before. Feel free to re-label, if I was wrong.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Comment by belm0** [comment]
> Hello, as you may have heard, we are https://github.com/grafana/grafana/discussions/73424. Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you

It's ridiculous, because it was originally a [github issue](https://github.com/grafana/grafana/issues/29480) that was moved to discussions.
 
The onus is completely on the Grafana organization to recognize that this is likely the case for many such "discussions", and summarily closing them when they were originally proper github issues is disingenuous.  @gelicia

**Comment by gelicia** [comment]
@belm0 The switching from issues to discussions, then back to issues is explained in the link about the move https://github.com/grafana/grafana/discussions/73424 We simply thought moving to discussions would be the best move for us, and then after a few years, noticed that it was more confusing for us and our community. We decided to move back.
Our criteria was that discussions that were older, and had a limited number of engagements would be closed. We hoped if the people passionate about the feature were still involved, they would make the issue again. Since 2017, a large number of discussions were no longer relevant because the feature was added without closing the discussion. As of right now, we plan on migrating 453 discussions back to issues. Thank you for your patience, and I hope you continue to participate in the Grafana community.
